organization/presentations.md

```diff
@@ -1,16 +1,19 @@
 ---
 title: "HSF Presentations"
-layout: default
+layout: plain
 ---
 
-# HSF Presentations
-
 ## 2021
 
+* [HEP Computing Challenges and the HSF](https://indico.physics.lbl.gov/event/1756/contributions/6247/), [Snowmass Small Experiments Workshop](https://indico.physics.lbl.gov/event/1756/), 16 November 2021, Elizabeth Sexton-Kennedy
```

Comment:
Maybe just the comment that in this case you did not add the direct link to the PDF?

Reply:
Right - which is what I tend to do when you can link to the "contribution", so it's deliberate. The direct slides link can be fragile if the slides get updated and the URL changes (albeit unlikely post facto, but still...).